app/services/vehicle_age_calculator.rb

```diff
@@ -0,0 +1,12 @@
+class VehicleAgeCalculator
+  def initialize(purchase_date, calculation_date)
+    @purchase_date = Date.parse(purchase_date)
+    @calculation_date = calculation_date
+  end
+
+  def in_months
+    months = (@calculation_date.year * 12 + @calculation_date.month) - (@purchase_date.year * 12 + @purchase_date.month)
```

Comment:
would something like `(@calculation_date - @purchase_date).months` work?

Reply:
I don't think so.  `#months` is a method on Fixnum.  Subtracting 1 data from another gives a Rational, representing the number of days.